frontend/src/components/App/Model/Modals/AddModelFileModal.tsx

```diff
@@ -88,7 +88,7 @@ class AddModelFileFormImpl extends React.Component<AddModelFileFormProps, AddMod
           validationSchema={AddModelFileSchema}
           onSubmit={this.onSubmit}
           onReset={this.onCancel}>
-          {({ errors, touched, setFieldValue, isSubmitting }) => (
+          {({ errors, touched, setFieldValue }) => (
```

Comment:
You don't need to rewrite here. You can write this like [this way](https://github.com/rekcurd/dashboard/blob/master/frontend/src/components/App/Models/ModelsDeleteForm.tsx#L25-L34).

Reply:
@keigohtr 
I changed here. https://github.com/rekcurd/dashboard/pull/108/commits/663319b30708cd63e68c75a780a86e19afa81358

Please review it again.